lab_1_classify_by_unigrams/main.py

```diff
@@ -34,6 +46,22 @@ def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
 
     In case of corrupt input arguments, None is returned
     """
+    if not isinstance(tokens, list):
+        return None
+    for token in tokens:
+        if not isinstance(token, str):
+            return None
+    freq = {}
+    prop = {}
+    total_tokens = len(tokens)
```

Comment:
don't create a new variable if you only use it once

Reply:
removed